crates/core/src/fnc/rand.rs

```diff
@@ -29,52 +29,60 @@ pub fn r#enum(mut args: Vec<Value>) -> Result<Value, Error> {
 	})
 }
 
+// TODO (Delskayn): Don't agree with the inclusive ranges in the functions here,
+// seems inconsistent with general use of ranges not including the upperbound.
+// These should probably all be exclusive.
+//
+// TODO (Delskayn): Switching of min and max if min > max is also inconsistent with rest of
+// functions and the range type. The functions should either return NONE or an error if the lowerbound
+// of the ranges here are larger then the upperbound.
```

Comment:
Agree with all of this 👍

Reply:
We should probably wait to implement these changes until we release another major version because changing it is a possibly breaking change.